app/controllers/compute_resources_vms_controller.rb

```diff
@@ -79,14 +79,7 @@ def console
     @compute_resource = find_compute_resource(:console_compute_resources_vms)
     @vm = find_vm
     @console = @compute_resource.console @vm.identity
-    @encrypt = case Setting[:websockets_encrypt]
-               when 'on'
-                 true
-               when 'off'
-                 false
-               else
```

Comment:
This is what the `auto` setting was supposed to do, however it seems like it would not honor `websockets_encrypt = off` as you pointed out.